### DISCUSSION TITLE
Reduce the accuracy required

### DISCUSSION THREAD

**Discussion by Peter230655** [csu-hmc/opty#268] [Q&A]
Is the a way in opty to reduce the accuracy for opty to be 'satisfied with the result'?
Reason for my question: I may have a nice example for examples-gallery, but it takes too long to run.
Better initial conditions do not seem to help much in this case.

**Thanks!**

**Comment by moorepants** [comment]
You can control any option for IPOPT, including solution tolerances.

**Reply by Peter230655** [reply]
**Thanks!**
But where would I find how to do it? I do not think, it is in opty documents?

**Reply by moorepants** [reply]
All IPOPT options are supported by opty. You have to read the IPOPT (and the cyipopt) documentation to see how to set options for IPOPT.

**Reply by Peter230655** [reply]
So, I would write: prob.add_option('acceptable_tol', 1.e-3) for example?
I found acceptable_tol in the Ipopt documentation.

**Reply by moorepants** [reply]
Yes, that is how you set options for IPOPT.

**Comment by Peter230655** [comment]
Thanks! Again I learned something! :-)
